chajda/tsquery/augments.py

```diff
@@ -115,6 +241,7 @@ def augments_fasttext(lang, word, config=Config(), n=5):
 from contextlib import contextmanager,redirect_stderr,redirect_stdout
 from os import devnull
 
+
```

Comment:
there's still a handful of minor formatting adjustments in the code like this, but that's fine